ITR/data/base_providers.py

```diff
@@ -63,14 +66,15 @@ def _get_projected_production(self, scope: EScope = EScope.S1S2) -> pd.DataFrame
 
         return df_bm
 
-    def get_company_projected_production(self, company_sector_region_info: pd.DataFrame) -> pd.DataFrame:
+    def get_company_projected_production(self, company_sector_region_info: pd.DataFrame, scope: EScope) -> pd.DataFrame:
         """
         get the projected productions for list of companies in ghg_scope12
         :param company_sector_region_info: DataFrame with at least the following columns :
         ColumnsConfig.COMPANY_ID, ColumnsConfig.GHG_SCOPE12, ColumnsConfig.SECTOR and ColumnsConfig.REGION
+        :param scope: benchmark scope for projections
```

Comment:
Ditto here: scope projections for production make no sense.

A long time ago, the code had significant confusion around naming where things that were emissions were labeled as production, and vice versa.  I fixed most of that, but believe this is a vestigial case.  Emissions and Emissions intensity have scopes.  Production, by itself, has no scope.

Reply:
Yes, agree. We will track this discussion in a different comment

Reply:
I have looked for, but not found, where we finalized the discussion about using a scope parameter for get_company_projected_production.  In any case, unless you have a really great example of where production output is somehow scope-dependent, I'm going to revert all these.  Note that in your test case (test_base_providers.py:test_get_projected_production where you pass EScope.S1S2 to one calculation and EScope.S3 to a second calculation), both result in the same answer (which you also remark in the comments).  Because there will never be any difference, it's just much cleaner and simpler to remove that parameter from that functional chain, which I will do in my subsequent merge.